docs/tips-and-tricks.md

```diff
@@ -21,6 +21,24 @@
+```
+
+If you would like to see the non-minified version: https://gist.github.com/nikitarevenco/f5470f16f562062cd88c9882aeb86611
```

Comment:
Could we either get this gist as a file on this repo, or @sgoudham can @Catppuccin-Holdings take ownership of this? Personally I prefer the first option.

Reply:
Ok, sure i have added it